Jenkinsfile

```diff
@@ -180,6 +209,10 @@ selectedArchitectures.each { suffix ->
         } else {
             cheribuildArgs.add('--cheribsd/no-debug-info')
         }
+
```

Comment:
```suggestion
```

Reply:
Do we also want the no- version in an else branch in case we change the default in cheribuild (which I imagine we will in the not too distant future)?

Reply:
I think if we change the default in cheribuild it will be the default kernel ABI though, which means that `--cheribsd/build-alternate-abi-kernels` will build the hybrid one as we want. Perhaps we should set `--cheribsd/default-kernel-abi hybrid` here to enforce that the selectedPurecapKernelArchitectures are applied correctly.

Reply:
I guess at that point we could just change the parameter name?